scripts/_infra_k8s.py

```
@@ -0,0 +1,349 @@
+        Result of the plan command.
+    """
+    args = ["plan", "-input=false"]
+    if var_file:
+        args.append(f"-var-file={var_file}")
+    return run_tofu(args, cwd)
+
+
+def tofu_apply(
+    cwd: Path,
+    var_file: Path | None = None,
+    auto_approve: bool = True,
```

Ruff (0.14.13)

Boolean-typed positional argument in function definition

(FBT001)